spec/models/message_spec.rb

```diff
@@ -43,6 +43,18 @@
 
     describe "#message_html" do
       it "creates rich link with proper link for article" do
+        link = "http://#{ApplicationConfig['APP_DOMAIN']}#{article.path}"
```

Comment:
Would it make sense to use our URL helper here?

```suggestion
        link = URL.article(article)
```